build.gradle

```diff
@@ -1,13 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext {
+        compose_version = '1.2.0-alpha05'
+        wear_compose_version = '1.0.0-alpha18'
```

Comment:
It looks like the latest Wear Compose stable release is `1.0.2`, and the latest alpha release is `1.1.0-alpha07`:
https://developer.android.com/jetpack/androidx/releases/wear-compose

Let's go with `1.1.0-alpha07` for now, and if we encounter problems we can fall back to the latest stable `1.0.2`.

Reply:
> 

Thanks, `1.0.2` is ok. `1.1.0-alpha07` has caused a lot of conflicts.